### PR TITLE
fix(matchers): toHaveStyle matcher throws TypeError error

### PR DESCRIPTION
**Pull request by dojchek**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x ] The commit message follows our guidelines: https://github.com/ngneat/spectator/blob/master/CONTRIBUTING.md#commit
- [x ] Tests for the changes have been added (for bug fixes / features)
- [x ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[x ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?
Fixing the bug described [in this issue](https://github.com/ngneat/spectator/issues/497)
Issue Number: 497

## What is the new behavior?
toHaveStyle throws error

## Does this PR introduce a breaking change?
```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: projects/spectator/src/lib/matchers.ts [hunk 1]
```diff
@@ -65,7 +65,7 @@ const hasCss = (el: HTMLElement, css: { [key: string]: string }) => {
         continue;
       }
 
-      if (trim($el.css(prop)) !== trim(value) && trim(el.style[prop]) !== trim(value)) {
+      if (trim($el.get(0).style[prop]) !== trim(value) && trim(el.style[prop]) !== trim(value)) {
```
Comment (Coffee-Tea) [inline projects/spectator/src/lib/matchers.ts:68]: this is the breaking thing!
it stopped working for styles from .css/.scss/.styl/etc files!

can we revert that?
it's not a fix for types it's change for the logic !

Comment (Coffee-Tea) [reply projects/spectator/src/lib/matchers.ts:68]: @dojchek could u pls fix that back

Comment (NetanelBasal) [reply projects/spectator/src/lib/matchers.ts:68]: @dojchek please handle this. You said this isn't a breaking change.

Comment (dojchek) [reply projects/spectator/src/lib/matchers.ts:68]: @NetanelBasal I will revert it. I didn't say it's not a breaking change, but I surely would not push it if I knew it was..
@Coffee-Tea Can you please provide the tests which makes this fail?
Sorry for not seeing this earlier.

Comment (dojchek) [reply projects/spectator/src/lib/matchers.ts:68]: I see that it has been already reverted.

**Pull request merged**